source/common/common/logger.cc

```diff
@@ -57,14 +57,14 @@ void DelegatingLogSink::log(const spdlog::details::log_msg& msg) {
   absl::ReleasableMutexLock lock(&format_mutex_);
   if (!formatter_) {
     lock.Release();
-    sink_->log(fmt::to_string(msg.raw));
+    sink_->log(absl::string_view(msg.raw.data(), msg.raw.size()));
```

Comment:
i would be surprised if spdlog knows about absl::string_view. Is this auto converting back to string? Do we need to use a string_view inside spdlog somewhere?

Reply:
Interestingly, there *is* a string_view inside spdlog, but it's not absl.  See https://github.com/gabime/spdlog/blob/f14a4c0b1848304dee929f99c49bb168de532d18/include/spdlog/common.h#L59

However in this case, DelegatingLogSink::sink_ is declared as:
```
  SinkDelegate* sink_{nullptr};
```

And SinkDelegate::log is declared as
```
  virtual void log(absl::string_view msg) PURE;
```